### ISSUE TITLE
plotIdiogram issues - SNPchip R package installation fails 

### ISSUE THREAD

**Issue by ngsdhbay** [gavinha/TitanCNA#88]
SNPchip package is unable to get installed as it's not available for R>4.0.0

If I go with R 3.6.3, GenomicRanges installation fails. What's the way here to get plotIdiogram working?

Error in plotIdiogram(chr, build = "hg19", unit = "bp", label.y = label.y,  :
  could not find function "plotIdiogram"

**Comment by pawel125** [comment]
The same issue here. Did you solve it in anyway?

**Comment by ngsdhbay** [comment]
Nope. I have to try using a package manager to see if that option works to keep different versions of R

**Comment by pawel125** [comment]
It worked for with the following conda environment:

```
name: titancna_env
channels:
  - conda-forge
  - bioconda
  - r
  - dranew
  - defaults
dependencies:
  - bioconductor-titancna
  - bioconductor-snpchip
  - bioconductor-bsgenome.hsapiens.ucsc.hg19
  - r-ichorcna
  - r-domc
  - hmmcopy_utils
  - pysam # not sure if necessary
  - r-ff=2.2.14
```

Before, I could not install SNPchip because of the higher ff version
